### PR TITLE
Feature/consume flow

### PR DESCRIPTION
**Pull request by jernejpregelj**
## Description

Consume flow WIP-PR.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [x] Follows the code style of this project.
- [x] Tests Cover Changes
- [x] Documentation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,18 +45,18 @@
...
     "react-vis": "^1.11.5",
-    "react-web3": "^1.2.0",
+    "react-web3-provider": "^1.1.1",
     "redux": "^4.0.1",
     "redux-devtools-extension": "^2.13.6",
     "redux-form": "^7.4.2",
     "redux-thunk": "^2.3.0",
     "secp256k1": "^3.5.2",
-    "web3": "^1.0.0-beta.36"
+    "web3": "1.0.0-beta.34"
```
Comment (H34D) [inline package.json]: this can be removed, squid brings its own web3 version

File: config/cloudStorage.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
     storageAccount: 'testocnfiles',
-    accessKey: 'DCNmb542DWbtkPf1lKz+WXii6Z50vScBvVJQMOy4XtG+bVIjHymbKm8iUZnSdlQlLRsrxlWaOwvAzbQmSm/oBw==',
+    accessKey: 's1wQNLo/on+vMcl20YuQGUY3BUU4+xcGqspiq4OXzdKy6FqfrR+MaIyWf1m1sWBxdbr0DommImHTi3alfQxprQ==',
```
Comment (H34D) [inline config/cloudStorage.js]: we should never store this info here

**Comment by jernejpregelj** [comment]
This branch needs review and bumping of squid-js when consuming asset is fixed and it theoretically should work.

File: package.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
   },
   "dependencies": {
     "@oceanprotocol/art": "^2.1.0",
-    "@oceanprotocol/squid": "^0.1.27",
+    "@oceanprotocol/squid": "^0.2.6",
```
Comment (jcortejoso) [inline package.json]: Released `0.2.7` with keeper-contracts `0.5.3`.

**Comment by jernejpregelj** [comment]
Just need to test it with https://github.com/oceanprotocol/brizo/pull/65 and we can merge.

**Comment by jernejpregelj** [comment]
@jcortejoso confirm changes so I can merge

**Comment by jernejpregelj** [comment]
This Pr includes verions for https://github.com/oceanprotocol/ocean/issues/263

**Pull request merged**